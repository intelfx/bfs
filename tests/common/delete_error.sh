cd "$TEST"

"$XTOUCH" -p foo/bar baz/qux
chmod -w foo
defer chmod +w foo

! invoke_bfs . -print -delete -print >"$OUT" || fail
sort_output
diff_output
