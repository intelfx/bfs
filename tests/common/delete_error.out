.
.
./baz
./baz
./baz/qux
./baz/qux
./foo
./foo/bar
